README.md

```diff
@@ -1,2 +1,23 @@
 # friday-projects
-An app to manage the awesomeness that is Friday Projects 
+An app to manage the awesomeness that is Friday Projects
+
+# Data Model
+The Data Model For This Project Consists of the Following Tables:
+
+* Photos
+  - URL
+  - Caption
+  - ProjectID
+
+* Projects
+  -Title
+  -Description
+  -RepoUrl
+  -Date
+
+* ProjectTeam
+  -ProjectID
+  -TeamMemberID
+
+* Team Member
+- Name
```

Comment:
It would be good to show the image in the README. Let's move the image into a `docs` folder (e.g. `/docs/datamodel.jpg`)